src/main/java/io/github/thebusybiscuit/sensibletoolbox/blocks/machines/BasicSolarCell.java

```diff
@@ -260,7 +260,7 @@ public int getTickRate() {
 
     private void drawPVLayer(Block b) {
```

Comment:
`@Nonnull` for the parameter Block here too

Reply:
Intelij says it not needed there

Reply:
`@Nonnull`